README.md

```diff
@@ -83,6 +83,8 @@ readonlyrest:
 
 readonlyrest:
     enable: true
+    verbosity: error # don't log successful request
```

Comment:
I've just checked verbosity implementation and I see a problem with it. Verbosity at 'readonlyrest' level was defined but not used. IMO this settings should be taken into consideration when no rule was matched. Moreover, verbosity functionality can be implemented without introducing new rule - when we treat verbosity as block setting, not specific rule, there is no need to mutate request context. I've changed the implementation to meet these criteria. 

But IMO naming is not so good. Current `check` function:
```public CompletableFuture<BlockExitResult> check(RequestContext rc) {
    logger.debug("checking request:" + rc.getId());
    return FuturesSequencer.runInSeqUntilConditionIsUndone(
        blocks.iterator(),
        block -> {
          rc.reset();
          return block.check(rc);
        },
        (block, checkResult) -> {
          if (checkResult.isMatch()) {
            if (Verbosity.INFO.equals(block.getVerbosity())) {
              logger.info("request: " + rc + " matched block: " + checkResult);
            }
            if (checkResult.getBlock().getPolicy().equals(BlockPolicy.ALLOW)) {
              rc.commit();
            }
            return true;
          } else {
            return false;
          }
        },
        nothing -> {
          if (Verbosity.INFO.equals(settings.getVerbosity())) {
            logger.info(ANSI_RED + " no block has matched, forbidding by default: " + rc + ANSI_RESET);
          }
          return BlockExitResult.noMatch();
        }
    );
  }

At the moment when verbosity level is set to INFO all matching is logged and no block match is logged also. When user sets verbosity to ERROR logging is disabled.

Maybe I missed sth, but verbosity values are not so descriptive (for me).